pkg/nordic_softdevice_ble/src/gnrc_nordic_ble_6lowpan.c

```diff
@@ -246,14 +246,20 @@ static gnrc_pktsnip_t *_netif_recv(gnrc_netif_t *netif)
 
 static void _netif_msg_handler(gnrc_netif_t *netif, msg_t *msg)
 {
+    netdev_t *dev = netif->dev;
     switch (msg->type) {
+        case NETDEV_MSG_TYPE_EVENT:
+            dev->driver->isr(dev);
+            break;
         case BLE_EVENT_RX_DONE:
             {
                 DEBUG("ble rx:\n");
                 _handle_raw_sixlowpan(msg->content.ptr);
                 ble_mac_busy_rx = 0;
                 break;
             }
+        default:
+            break;
     }
```

Comment:
How about just calling `gnrc_netif_msg_handler_netdev()` after the module's messages are handled in cases like this?

Reply:
I can do that

Reply:
for that one is fine. For gomach and lwmac I would keep it as it is, since there's a check in the end that might delay ISR processing.

Reply:
You could also put it in the beginning.. at the end was just a suggestion. I just think that the same action should use the same function.

Reply:
done!